recipes/westpa/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "westpa" %}
+{% set version = "2017.10" %}
+{% set sha256 = "3129dd1d651c4fe542599f2d5ecdbff09e5d704ea8c97c8ae7dca7b0bd4598a6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/westpa/westpa/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: './setup.sh && cp -r $SRC_DIR $PREFIX'
+  skip: True  # [win or py3k]
+
+requirements:
+  build:
+    - python
+    - toolchain
+    - cython
+    - numpy 1.11.*
+  run:
+    - python
+    - numpy >=1.11
+    - scipy 1.0.*
+    - h5py 2.7.*
+    - pyyaml 3.12
+    - pyzmq 16.0.*
+
+test:
+  commands:
+    - "source ${PREFIX}/${PKG_NAME}-${PKG_VERSION}/westpa.sh"
+    - "source ${WEST_ROOT}/bin/w_env"
```

Comment:
If we are needing to `source` things, maybe we should use an `activate.sh` script. The `toolchain` is a good example to pull from here.

Reply:
An `activate.sh` seems like a good fit for what `source` is functionally doing here. My only concern is that using `source` is how all of the documentation suggests doing things historically before we ever attempted to install via something like conda. For consistency (at least in the short term), I'm wondering if we should stick with the current scheme if there isn't any issue other than verbosity on the user end of things.

Reply:
I don't have strong opinions here. These are more suggestions that I think would keep the recipe maintainable, follow more common `conda` recipe strategies, and hopefully minimize the work needed by the end user.

FWIW this can probably be done in a way where running this separate `source` step again is just a no-op.